client/lib/domains/resolve-domain-status.js

```diff
@@ -252,18 +345,51 @@ export function resolveDomainStatus(
 					status: translate( 'Expiring soon' ),
 					icon: 'info',
 					listStatusText: expiresMessage,
+					noticeText: expiresMessage,
 					listStatusClass: 'warning',
 					listStatusWeight: 800,
 				};
 			}
 
 			if ( isRecentlyRegistered( domain.registrationDate ) ) {
+				let noticeText;
+				if ( domain.isPrimary ) {
+					noticeText = translate(
+						'We are setting up your domain. It should start working immediately, but may be unreliable during the first 30 minutes. If you are unable to access your site at %(domainName)s, try setting the primary domain to a domain you know is working. {{learnMore}}Learn more{{/learnMore}} about setting the primary domain, or try {{try}}%(domainName)s{{/try}} now.',
+						{
+							args: {
+								domainName: domain.name,
+							},
+							components: {
+								learnMore: (
+									<a href={ SETTING_PRIMARY_DOMAIN } rel="noopener noreferrer" target="_blank" />
+								),
+								try: (
+									<a href={ `http://${ domain.name }` } rel="noopener noreferrer" target="_blank" />
+								),
+							},
+						}
+					);
+				} else {
+					noticeText = translate(
+						'We are setting up your domain. It should start working immediately, but may be unreliable during the first 30 minutes. {{learnMore}}Learn more{{/learnMore}}',
```

Comment:
I'm not sure the "learn more" link is really useful here, without any context provided (I mean, no explanation about that it's not currently set as primary).

Reply:
Alright, I'll remove this one for now and we can work on it in a follow up PR.